src/steps/index.test.ts

```diff
@@ -12,7 +16,20 @@ const integrationConfig: IntegrationConfig = {
   oktaApiKey: process.env.OKTA_API_KEY || DEFAULT_API_KEY,
 };
 
+jest.setTimeout(1000 * 60 * 1);
+let recording: Recording;
+afterEach(async () => {
+  await recording.stop();
+});
+
 test('should collect data', async () => {
+  recording = setupOktaRecording({
+    directory: __dirname,
+    name: 'steps',
+    redactedRequestHeaders: ['Authorization'],
+    redactedResponseHeaders: ['set-cookie', 'public-key-pins-report-only'],
```

Comment:
💯

Reply:
Would it make sense to add this to the `setupOktaRecording` code as defaults for all Okta requests?

Reply:
Yeah, makes sense to me. Did that.